### ISSUE TITLE
custom_name is blank

### ISSUE THREAD

**Issue by lozzd** [USA-RedDragon/python-google-sdm#1]
First - thanks for this. Finally can get some data back from my devices in a sane way. 

Have you got the custom names working? Looking through the code it seems like it's matching the API, but I'm getting blank names for all my devices instead of.... well, anything would be great. So not sure which thermostat is which (I've got two)

Using 
```    
devices = api.get_devices()
    for device in devices:
        print(device)
        print(device.get_info())
```

Returns: 

```
sdm.devices.types.THERMOSTAT
sdm.devices.traits.Info{'custom_name': ''}
<snip>
sdm.devices.types.THERMOSTAT
sdm.devices.traits.Info{'custom_name': ''}
```

Thanks!

**Comment by USA-RedDragon** [comment]
Yeah, I had a similar situation when testing their API. Apparently even though a thermostat is named appropriately in consumer applications like Nest's app, in Google's API I believe the custom name refers to a name assigned in the Google Home app. I'm not 100% certain on that though
